README.md

```diff
@@ -27,82 +28,112 @@ Install `yarn` globally:
 npm install --global yarn
 ```
 
-## Set environment variables
+## Add sf-toolkit package
+
+Create `package.json` by running the following command and run through the prompts:
+
+```
+yarn init
+```
+
+Install `sf-toolkit` as a dependency of your project:
 
-You can either add the environment variables locally in the file so you only need to add them once, or define them directly in the terminal. 
+```
+yarn add https://github.com/silverfin/sf-toolkit.git
+```
 
-### Local file 
+## Add scripts to packages.json
 
-Add a .env file in the root directory:
+You can add this `scripts` section to the `packages.json`, where we define `silverfin` as an alias to call our CLI.
 
 ```
-touch .env
+  "scripts": {
+    "silverfin": "node ./node_modules/sf_toolkit/bin/cli.js"
+  },
 ```
 
-Add the following variables in  .env:
+## Add your API credentials as environmental variables
+
+You could use a new `.env` local file, or add them to an existing file in your system (e.g. `~/.bash-profile` or `~/.bashrc`)
 
 ```
-SF_FIRM_ID="your firm ID"
-SF_ACCESS_TOKEN="your access token"
-NODE_ENV="development"
+export SF_API_CLIENT_ID=...
+export SF_API_SECRET=...
 ```
 
-### Linux / Mac terminal
+If you are only using one firm, you could set it up as an environmental variable. That way, you won't need to pass the `--firm <firmid>` option every time you run a command.
 
 ```
-export SF_FIRM_ID=<firm_id>
-export SF_ACCESS_TOKEN=<access_token>
+export SF_FIRM_ID=...
```

Comment:
Okay for the first iteration, but you can also include a command in your package to set the firm/context they want to interact with, maybe `yarn silverfin -f firm_id_here` or `yarn silverfin -firm firm_id_here` which stores the current firm moving forward, it's more flexible than setting an env var or entering the firm id with every command